backend/app/Http/Controllers/Timeline/ShowAllTimelineController.php

```diff
@@ -25,11 +25,11 @@ public function __invoke(): View|Factory
          */
         $timelines = UserTimeline::with('User:id,name')
             ->orderBy('start_date', 'desc')
-            ->limit(100)
+            ->limit('400')
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter #1 $value of method Illuminate\Database\Eloquent\Builder<App\Models\UserTimeline>::limit() expects int, string given.